piston/configuration/choose_config.py

```diff
@@ -0,0 +1,13 @@
+import random
+from typing import Any
+
+from piston.utilities.constants import console
+
+
+def choose_config(config: Any) -> Any:
+    """Chooses a random value from a list of config options or returns the config if it is not in a list."""
+    if type(config) is list:
```

Comment:
You can use `isinstance()` here,
```suggestion
    if isinstance(config, list):
```